src/SFA.DAS.EmployerApprenticeshipsService.Infrastructure/Services/CommitmentsService.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using SFA.DAS.Commitments.Api.Client;
+using SFA.DAS.EAS.Domain.Interfaces;
+
+namespace SFA.DAS.EAS.Infrastructure.Services
+{
+    public class CommitmentsService : ICommitmentsService
+    {
+        private readonly ICommitmentsApi _commitmentsApi;
+
+        public CommitmentsService(ICommitmentsApi commitmentsApi)
```

Comment:
Ok cool - does seem slightly redundant but seems to be the way they've gone elsewhere in the solution